e2e/tests/home/homepage.spec.ts

```diff
@@ -1,51 +1,142 @@
-import GoogleLoginPage from '../../app/google-login';
-import Home from '../../app/home';
-import WorkspacePage from '../../app/workspace-page';
-import PuppeteerLaunch from '../../services/puppeteer-launch';
+import Link from '../../app/aou-elements/Link';
+import WebElement from '../../app/aou-elements/WebElement';
+import GoogleLoginPage from '../../app/GoogleLoginPage';
+import HomePage, {FIELD_LABEL as editPageFieldLabel} from '../../app/HomePage';
+import WorkspaceCard from '../../app/WorkspaceCard';
+import WorkspaceEditPage from '../../app/WorkspaceEditPage';
+import WorkspacesPage from '../../app/WorkspacesPage';
+import launchBrowser from '../../driver/puppeteer-launch';
 
 jest.setTimeout(60 * 1000);
 
-const configs = require('../../resources/config');
+const configs = require('../../resources/workbench-config');
 
-describe.skip('Home page tests:', () => {
+describe('Home', () => {
   let browser;
   let page;
-  let cookies;
 
   beforeAll(async () => {
-    browser = await PuppeteerLaunch();
+    browser = await launchBrowser();
   });
 
   beforeEach(async () => {
     page = await browser.newPage();
     await page.setUserAgent(configs.puppeteerUserAgent);
-    await page.setDefaultNavigationTimeout(60000);
   });
 
   afterEach(async () => {
     await page.close();
+    await page.waitFor(1000);
   });
 
   afterAll(async () => {
     await browser.close();
   });
 
-  test('Homepage is the landing page after sign in', async () => {
+  test('Homepage is the landing page after Sign In', async () => {
+    // Enable networks requests inspection
+    await page.setRequestInterception(true);
+
+    // Following network requests expected to happen
+    const targetRequestsUrls = [
```

Comment:
What's the rationale for checking this? Isn't this a bit of a blackbox implementation detail? I would think in an integration test we would prefer to just check whether everything got rendered.

Reply:
couple reason. first, I wanted to try out code to learn how to intercept network requests in test. second, I want to grab workspaces found in response json and verify against displayed workspaces loaded in UI. I haven't been able to find time now to implement part two to make this test do more, but I will.